### PR TITLE
chore: enable announcement bar, explain brownout

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Enable the announcement bar in config again
- Write announcement bar text for brownout

## Context:

Related discussion: 

- https://github.com/renovatebot/renovate/discussions/20613

### Preview image:

preview-announcement-bar-brownout

### CODE REVIEW

**Comment by rarkins** [comment]
The announcement bar is a big big. Wondering if we can reduce it, as it only applies to <0.1% of installs

**Review by viceice — APPROVED** [review]

File: overrides/main.html [hunk 1]
```diff
@@ -6,5 +6,16 @@
 {% extends "base.html" %}
 
 {% block announce %}
-We released two new major versions of Renovate. Read the <a href="https://docs.renovatebot.com/release-notes-for-major-versions/" target="_blank">Release notes for major versions of Renovate</a> section of the docs to learn what's changed.
+<p>If you're using the GitHub-hosted app version of Renovate, <em>and</em> you invited the <code>renovate</code> npm user into your npm organization then keep reading this message.</p>
```
Comment (HonkingGoose) [inline overrides/main.html:9]:
```suggestion
<p>If you're using the GitHub-hosted app version of Renovate, <em>and</em> you invited the <code>renovate</code> npm user into your npm organization then keep reading.</p>
```

File: overrides/main.html [hunk 2]
```diff
@@ -6,5 +6,16 @@
...
+
+<p>
+  We are phasing out using the <code>renovate</code> npm user for private packages.
+  We are deliberately skipping provisioning this npm token for 20 percent of the time (brownout).
+  This means that Renovate will autoclose some pull requests, which will then be re-opened the next time Renovate runs with the token.
```
Comment (HonkingGoose) [inline overrides/main.html:14]:
```suggestion
  This means that Renovate automatically closes some pull requests, which are re-opened the next time Renovate runs with the token.
```

File: overrides/main.html [hunk 3]
```diff
@@ -6,5 +6,16 @@
...
+</p>
+
+<p>
+  If you're seeing this behavior, it means you need to set up your own service user token.
```
Comment (HonkingGoose) [inline overrides/main.html:18]:
```suggestion
  If you're seeing this behavior, you need to set up your own service user token.
```

**Review by HonkingGoose — COMMENTED** [review]
Here are some suggestions to reduce the wall of text a bit.

File: overrides/main.html [hunk 4]
```diff
@@ -6,5 +6,16 @@
...
 {% block announce %}
-We released two new major versions of Renovate. Read the <a href="https://docs.renovatebot.com/release-notes-for-major-versions/" target="_blank">Release notes for major versions of Renovate</a> section of the docs to learn what's changed.
+<p>If you're using the GitHub-hosted app version of Renovate, <em>and</em> you invited the <code>renovate</code> npm user into your npm organization then keep reading.</p>
+
+<p>
+  We are phasing out using the <code>renovate</code> npm user for private packages.
+  We are deliberately skipping provisioning this npm token for 20 percent of the time (brownout).
+  This means that Renovate will autoclose some pull requests, which will then be re-opened the next time Renovate runs with the token.
+</p>
+
```
Comment (rarkins) [inline overrides/main.html:16]:
```suggestion
```

File: overrides/main.html [hunk 5]
```diff
@@ -6,5 +6,16 @@
...
+<p>
+  If you're seeing this behavior, it means you need to set up your own service user token.
```
Comment (rarkins) [inline overrides/main.html:18]:
```suggestion
  We are phasing out use of the `renovate` npm user, which affects about 10 early users of the Renovate App on github.com.
```

**Review by rarkins — CHANGES_REQUESTED** [review]
Let's keep it short and simple

**Review by rarkins — APPROVED** [review]

**Pull request merged**